test/unit/org/apache/cassandra/LogbackStatusListener.java

```diff
@@ -475,9 +479,9 @@ public synchronized void onStart(LoggerContext loggerContext)
                 Logger stdoutLogger = LoggerFactory.getLogger("stdout");
                 Logger stderrLogger = LoggerFactory.getLogger("stderr");
 
-                replacementOut = wrapLogger(stdoutLogger, originalOut, "sun.stdout.encoding", false);
```

Comment:
I think it would be better to keep these constants matching their names as we already have a lot of constants with `JAVA_` prefix. So `SUN_STDOUT_ENCODING` looks better according to everything we already have, WDYT? I have renamed it.

Reply:
I like  how it is done now.